lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -12,6 +12,8 @@
 #include "mlir/Support/LogicalResult.h"
 #include "mlir/Transforms/DialectConversion.h"
 #include "llvm/Support/Casting.h"
+#include <cassert>
+#include <cstdint>
```

Comment:
Check if we really need these and sort according to style guide: https://llvm.org/docs/CodingStandards.html#include-style